armi/reactor/tests/test_reactors.py

```diff
@@ -200,6 +208,16 @@ def loadTestReactor(
     return o, o.r
 
 
+def reduceTestReactorRings(r, cs, maxNumRings):
+    """Helper method for the test reactor above
+    The goal is to reduce the size of the reactor for tests that don't neeed
+    such a large reactor, and would run much faster with a smaller one
+    """
+    if 9 > maxNumRings > 1:
+        for ring in range(9, maxNumRings, -1):
+            r.core.removeAssembliesInRing(ring, cs)
+
```

Comment:
I get that this is a method that won't be touched too often, but do we want to make it a little more robust and general?

For instance:
- Telling the user when it doesn't do anything because the specified `maxNumRings` is out of range
- Ensuring that the upper limit is in line with the test reactor in the module, instead of hard-coding a value of `9` into this method

Reply:
Agreed.

The reason it looks like this was because it was originally INSIDE the `loadTestRector()` method. Which was much easier to keep it clean. Unfortunately, it can't live there because that breaks all the tests when they are run in parallel.

Sure though.  5 minutes

Reply:
done